### ISSUE TITLE
Game crashed by large amount of dead entities

### ISSUE THREAD

**Issue by 1oglop1** [jaredlll08/Clumps#146]
Hi I built this farm https://www.youtube.com/watch?v=Ln-F7y669nk
and it crashed the client when all the slimes died.
version: Fabric-1.21-18.0.0.1

```
Process crashed with exit code -1073740791 (0xffffffffc0000409).
Below is an analysis of the exit code. THIS MAY BE INCORRECT AND SHOULD BE TAKEN WITH A GRAIN OF SALT!
System exit code name: STATUS_STACK_BUFFER_OVERRUN
System exit code description: The system detected an overrun of a stack-based buffer in this application. This overrun could potentially allow a malicious user to gain control of this application.
```

<details><summary>Modlist</summary>

```       [main/INFO]: Loading 93 mods:
	- architectury 13.0.6
	- armorposer 6.0.4
	- carpet 1.4.147+v240613
	- cloth-config 15.0.128
	   \-- cloth-basic-math 0.6.1
	- clumps 18.0.0.1
	- controlling 18.0.1
	- coordfinder 1.21-1.1.0
	   |-- admiral 0.4.4+1.20.6
	   \-- fabric-api-base 0.4.42+6573ed8c90
	- entity_model_features 2.1.3
	- entity_texture_features 6.1.3
	   \-- org_apache_httpcomponents_httpmime 4.5.10
	- entityculling 1.6.6
	- fabric-api 0.100.7+1.21
	   |-- fabric-api-lookup-api-v1 1.6.67+b5597344d1
	   |-- fabric-biome-api-v1 13.0.29+5bd9f1bcd1
	   |-- fabric-block-api-v1 1.0.22+0af3f5a7d1
	   |-- fabric-block-view-api-v2 1.0.10+6573ed8cd1
	   |-- fabric-blockrenderlayer-v1 1.1.52+0af3f5a7d1
	   |-- fabric-client-tags-api-v1 1.1.15+6573ed8cd1
	   |-- fabric-command-api-v1 1.2.49+f71b366fd1
	   |-- fabric-command-api-v2 2.2.28+6ced4dd9d1
	   |-- fabric-commands-v0 0.2.66+df3654b3d1
	   |-- fabric-content-registries-v0 8.0.15+b5597344d1
	   |-- fabric-convention-tags-v1 2.0.18+7f945d5bd1
	   |-- fabric-convention-tags-v2 2.4.2+c111832ad1
	   |-- fabric-crash-report-info-v1 0.2.29+0af3f5a7d1
	   |-- fabric-data-attachment-api-v1 1.1.25+6a6dfa19d1
	   |-- fabric-data-generation-api-v1 20.2.12+16c4ae25d1
	   |-- fabric-dimensions-v1 4.0.0+6fc22b99d1
	   |-- fabric-entity-events-v1 1.6.12+6fc22b99d1
	   |-- fabric-events-interaction-v0 0.7.10+e633f883d1
	   |-- fabric-game-rule-api-v1 1.0.53+6ced4dd9d1
	   |-- fabric-item-api-v1 11.0.0+afdfc921d1
	   |-- fabric-item-group-api-v1 4.1.3+78017270d1
	   |-- fabric-key-binding-api-v1 1.0.47+0af3f5a7d1
	   |-- fabric-keybindings-v0 0.2.45+df3654b3d1
	   |-- fabric-lifecycle-events-v1 2.3.11+8f3583aed1
	   |-- fabric-loot-api-v2 3.0.13+3f89f5a5d1
	   |-- fabric-loot-api-v3 1.0.1+3f89f5a5d1
	   |-- fabric-message-api-v1 6.0.13+6573ed8cd1
	   |-- fabric-model-loading-api-v1 2.0.0+fe474d6bd1
	   |-- fabric-networking-api-v1 4.2.0+ab7edbacd1
	   |-- fabric-object-builder-api-v1 15.1.13+d1321076d1
	   |-- fabric-particles-v1 4.0.2+6573ed8cd1
	   |-- fabric-recipe-api-v1 5.0.10+65089712d1
	   |-- fabric-registry-sync-v0 5.0.25+5bd9f1bcd1
	   |-- fabric-renderer-api-v1 3.3.0+0ae0b97dd1
	   |-- fabric-renderer-indigo 1.6.5+48fb1586d1
	   |-- fabric-renderer-registries-v1 3.2.68+df3654b3d1
	   |-- fabric-rendering-data-attachment-v1 0.3.48+73761d2ed1
	   |-- fabric-rendering-fluids-v1 3.1.6+b5597344d1
	   |-- fabric-rendering-v0 1.1.71+df3654b3d1
	   |-- fabric-rendering-v1 5.0.5+df16efd0d1
	   |-- fabric-resource-conditions-api-v1 4.2.1+d153f344d1
	   |-- fabric-resource-loader-v0 1.2.0+7e687b32d1
	   |-- fabric-screen-api-v1 2.0.24+b5597344d1
	   |-- fabric-screen-handler-api-v1 1.3.82+b5597344d1
	   |-- fabric-sound-api-v1 1.0.23+6573ed8cd1
	   |-- fabric-transfer-api-v1 5.1.15+3dccd343d1
	   \-- fabric-transitive-access-wideners-v1 6.0.12+6573ed8cd1
	- fabric-language-kotlin 1.11.0+kotlin.2.0.0
	   |-- org_jetbrains_kotlin_kotlin-reflect 2.0.0
	   |-- org_jetbrains_kotlin_kotlin-stdlib 2.0.0
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk7 2.0.0
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk8 2.0.0
	   |-- org_jetbrains_kotlinx_atomicfu-jvm 0.24.0
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm 1.8.1
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-jdk8 1.8.1
	   |-- org_jetbrains_kotlinx_kotlinx-datetime-jvm 0.6.0
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm 1.6.3
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-core-jvm 1.6.3
	   \-- org_jetbrains_kotlinx_kotlinx-serialization-json-jvm 1.6.3
	- fabricloader 0.15.11
	   \-- mixinextras 0.3.5
	- inventoryprofilesnext 2.0.2
	- java 21
	- libipn 6.0.0
	- litematica 0.18.999-sakura.28
	- lithium 0.12.7
	- malilib 0.20.0
	   |-- fabric-networking-api-v1 4.2.0+ab7edbacd1
	   \-- fabric-resource-loader-v0 1.2.0+7e687b32d1
	- minecraft 1.21
	- minihud 0.31.999-sakura.13
	- modmenu 11.0.1
	   |-- fabric-api-base 0.4.42+6573ed8c90
	   \-- placeholder-api 2.4.0-pre.2+1.21
	- nvidium 0.2.9-beta
	- searchables 1.0.1
	- shulkerplus 1.2.2
	- sodium 0.5.11+mc1.21
	   \-- fabric-api-base 0.4.42+6573ed8c90
	- tcdcommons 3.12.2+fabric-1.21
	- tweakeroo 0.20.999-sakura.20
	- wi_zoom 1.5-MC1.21```

</details>

**Comment by jaredlll08** [comment]
I'm not sure why you to believe this is clumps compared to something else, the crash doesn't mention my mod, or any mod at all.

Looking the error up, leads me to believe that something on your system is corrupt, either windows itself or potentially your graphics drivers, there should be a crash log from the jvm somewhere, which has more information.

Unfortunately there is nothing I can do on my side to fix this issue.

Here are some of the links I found that could help you fix it:
https://bugs.mojang.com/browse/MC-243616?attachmentViewMode=list

https://answers.microsoft.com/en-us/windows/forum/all/solvederror-system-detected-an-overrun-of-a-stack/23ca258b-480a-448f-bd52-78591ea34cac

https://www.reddit.com/r/runescape/s/L4g2OouKMk

**Comment by 1oglop1** [comment]
Thanks for the quick response! 

I'll take a look at the drivers. The thing is that I did a binary search of the suspicious mods and Clumps was in the group that caused the problem. When I disabled Clumps, the thing worked. 
But later on, it crashed again even without Clumps, so I disabled all mods and it worked just fine.
It was also related to the number of entities.

**Update**: after updating the graphics drivers the issue is fixed.
